NEXT-18240 - App feedback requests (tab and modal) are signed (#438)
shopware/docs: guides/plugins/apps/administration/add-custom-action-button.md

* `entity`: Here you define which entity you're working on.
* `view`: `detail`or `list`;  to set the view the button should be added to. Currently, you can choose between detail and listing view.

When the user clicks on the action button your app receives a request similar to the one generated by a [webhook](../app-base-guide.md#webhooks).

> The main difference is that it contains the name of the entity and an array of ids that the user selected \(or an array containing just a single id in case the detail page\).

Reviewer: ```suggestion
The main difference is that it contains the name of the entity and an array of ids that the user selected \(or an array containing just a single id if the request was sent from a detail page\).
```

This feature was added in Shopware 6.4.3.0, previous versions will ignore the response content.
{% endhint %}

{% hint style="info" %}

> Starting from Shopware version 6.4.8.0, the requests of the [tab](#opening-a-new-tab-for-the-user) and [custom modal](#open-a-custom-modal) will have the following additional query parameters:

Reviewer 2: ```suggestion
Starting from Shopware version 6.4.8.0, the requests of the [tab](#opening-a-new-tab-for-the-user) and [custom modal](#open-a-custom-modal) have the following additional query parameters:
```

Reviewer 3: Shouldn't it be "[...] and custom modal **HAS** the following additional [...]"?

> * shop-id

Author: I hope the hint boxes allow listings, I cannot generate a preview 👀
